src/style.css

```diff
@@ -0,0 +1,108 @@
+body {
+  margin: 0;
+}
+
+.card {
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  font-weight: 400;
+  width: 200px;
+  height: 403px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.product-image {
+  margin: 0 auto;
+  margin-bottom: 45px;
+  margin-top: 30px;
+}
+
+.description {
+  display: inline;
+  margin-left: 15px;
+  margin-right: 15px;
+  
+}
+
+.info {
+  font-size: 12px;
+  font-weight: 500;
+  margin-bottom: 5px;
+  line-height: 15px;
+}
+
+.product-code {
+  margin-bottom: 15px;
+  
+}
+
+.stars,
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 15px;
+  margin-right: 15px;
+}
+
+.stars {
+  margin: 0;
+}
+
+.stars div {
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice